components/shared/searchFilters/SearchFilterBar.tsx

```diff
@@ -0,0 +1,36 @@
+"use client"
+
+import React, { useState } from "react"
+
+import { SearchFacetsQuery } from "@/lib/graphql/generated/fbi/graphql"
+import SearchFiltersColumn from "./SearchFiltersColumn"
+
+type SearchFilterBarProps = {
+  facets: SearchFacetsQuery["search"]["facets"]
```

Comment:
Don't we have a fragment for the facets. Otherwise I think we should create one and use that directly.

Reply:
Great thought!